apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -800,7 +811,7 @@ static void fs_libc_dirent_telldir(void)
 
 	/* Nagative case with invalid argument, NULL stream. It will return (off_t)-1 */
 	res = telldir(NULL);
-	TC_ASSERT_EQ("telldir", res, (off_t)-1);
+	TC_ASSERT_EQ("telldir", res, (off_t) - 1);
```

Comment:
Please revert this line. Previous one is right.

Reply:
Done. 
Sorry. This change was done by the formatter tool.